### ISSUE TITLE
No Crosshair MacOS

### ISSUE THREAD

**Issue by namedtoaster** [Hopson97/MineCraft-One-Week-Challenge#133]
The game runs great on my mac book BUT...the crosshair isn't showing and the mouse is _almost_ useless. By that I mean the mouse stops on the edges of the screen and therefore I can't rotate the view any more. Perhaps this is because I'm just using the trackpad? I would troubleshoot, but I've been spending the past few hours just reviewing the code and should probably get some z's.

Goodnight! 😴

**Comment by w-A-L-L-e** [comment]
Yes indeed it renders and you can walk with awsd  on mac also even fullscreen works if you edit config.txt and put it to 13 you press enter for fullscreen mode. But mouse is not working at all unfurtunately it jitters a little when you move the mouse but you cant really look around because of the bug it's unplayable. Same happens with trackpad and regular mouse ...
